doc/lua_api.md

```diff
@@ -564,17 +564,22 @@ Example:
 
     default_cobble.png^[crack:10:1
 
-#### `[combine:<w>x<h>:<x1>,<y1>=<file1>:<x2>,<y2>=<file2>:...`
-
-* `<w>`: width
-* `<h>`: height
-* `<x>`: x position, negative numbers allowed
-* `<y>`: y position, negative numbers allowed
-* `<file>`: texture to combine
+#### `[combine:<w>x<h>:<x1>,<y1>,<w1>=<file1>:<x2>,<y2>,<w2>=<file2>:...`
 
 Creates a texture of size `<w>` times `<h>` and blits the listed files to their
 specified coordinates.
 
+Note: the output texture size may vary if one or more of `<wN>` is provided.
```

Comment:
so the output will not be `<w> x <h>`?
I mean that's what it says but somehow I expected this to work differently

Reply:
That's the goal, yes. This way, modders can craft the texture modifier using their default texture pack resolution specified by `<wN>`. If the client's textures happen to have a higher resolution, the texture modifier will work in a larger scale to perform lossless combining. Hence this sentence.

Reply:
Ah, I see. Conceptually this would be much better in a new modifier designed from scratch, but the partial backwards compat is compelling.